irksome/stage_value.py

```diff
@@ -236,6 +239,22 @@ def _update_general(self):
         self.update_solver.solve()
         self.u0.assign(self.unew)
 
+    def _update_collocation(self):
+        # Use the terminal value of the collocation polynomial to update the solution. Note: collocation update is only implemented for constant-in-time boundary conditions.
+        # TODO: create an assertion to check for constant-in-time boundary conditions.
+        nodes = numpy.insert(self.butcher_tableau.c, 0, 0.0)
+
+        assert isinstance(self.butcher_tableau, CollocationButcherTableau), "Need a collocation method for collocation update"
+        assert (self.basis_type is None or self.basis_type == "Lagrange"), "Collocation update requires the Lagrange form of the collocation polynomial"
+        assert (len(set(nodes)) == self.butcher_tableau.num_stages + 1), "Need a non-confluent collocation method to use collocation update"
+
+        lag_basis = LagrangePolynomialSet(ufc_simplex(1), nodes)
+        vander = lag_basis.tabulate(numpy.array([[1.0]]), 0)[(0,)].flatten()
```

Comment:
I _think_ this also works
```suggestion
        vander = lag_basis.tabulate((1.0,)))[(0,)]
```

Reply:
Ah, indeed. Much cleaner.